docs/connecting/watching-assets.mdx

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
I'll remove/replace this once https://github.com/valora-inc/wallet/pull/3514 merges.